kubernetes-manifests/recommendationservice.yaml

```diff
@@ -61,6 +61,8 @@ spec:
           value: "productcatalogservice:3550"
         - name: DISABLE_PROFILER
           value: "1"
+        - name: OTEL_SERVICE_NAME
+          value: "recommendationservice"
         resources:
           requests:
             cpu: 100m
```

Comment:
**Question:** What do you think about moving the `/kubernetes-manifests/*.yaml` changes to the [`google-cloud-operations` Kustomize component](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/kustomize/components/google-cloud-operations/kustomization.yaml)?

(It's harmless to set `OTEL_SERVICE_NAME` by default, but I want to check with you if it makes more sense in that Kustomize component.)

Reply:
This will also help you avoid waiting for me to make a new release of Online Boutique. :)

Reply:
This change was not originated by Online Boutique :slightly_smiling_face: . But, I agree with your point that it will be cleaner to set it in the kustomize component.